src/183/test/kotlin/org/rust/ide/runAnything/CargoRunAnythingProviderTest.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.runAnything
+
+import com.intellij.openapi.actionSystem.DataContext
+import com.intellij.openapi.actionSystem.impl.SimpleDataContext
+import org.rust.EmptyDescriptor
+import org.rust.ProjectDescriptor
+import org.rust.RsTestBase
+
+class CargoRunAnythingProviderTest : RsTestBase() {
+    private lateinit var provider: CargoRunAnythingProvider
+    private lateinit var dataContext: DataContext
+
+    override fun setUp() {
+        super.setUp()
+        provider = CargoRunAnythingProvider()
```

Comment:
Looks like it doesn't have any state and can be initialized while `CargoRunAnythingProviderTest` object creation